samples/bluetooth/mesh/sensor_client/README.rst

```diff
@@ -80,6 +81,12 @@ Buttons:
    Can be used to input the OOB authentication value during provisioning.
    All buttons have the same functionality during the provisioning procedure.
 
+Button 1:
+   Sends a setting get message (after the provisioning procedure is finished).
+
+Button 2:
```

Comment:
Could be worth mentioning that it will toggle between the different ranges.

Reply:
I have :)